mmaction/apis/test.py

```diff
@@ -131,6 +131,7 @@ def collect_results_cpu(result_part, size, tmpdir=None):  # noqa: F811
             dist.broadcast(dir_tensor, 0)
             tmpdir = dir_tensor.cpu().numpy().tobytes().decode().rstrip()
         else:
+            tmpdir = osp.join('.dist_test', tmpdir)
```

Comment:
Wrong order of join?

Reply:
correct it to `osp.join(tmpdir, '.dist_test')`